cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java

```diff
@@ -351,7 +351,13 @@ public void onSuccess(Object result) {
                         setNativeMessageView(jsonResult.getJSONObject("msgJSON"));
                         showView(nativeView);
                     } else if(jsonResult.has("url") && !jsonResult.isNull("url")){
-                        loadConsentUI(jsonResult.getString("url"));
+                        String url = jsonResult.getString("url");
```

Comment:
This change is not related to this ticket, is it?

Reply:
Yes it is related.